shared/common-adapters/markdown.desktop.js

```diff
@@ -39,12 +39,12 @@ const codeSnippetBlockStyle = {
   paddingTop: globalMargins.xtiny,
 }
 
-const textBlockStyle = {display: 'block', ...wrapStyle, color: undefined, fontWeight: undefined}
-const linkStyle = {...wrapStyle, fontWeight: undefined}
-const neutralPreviewStyle = {color: undefined, fontWeight: undefined}
-const boldStyle = {...wrapStyle, color: undefined}
-const italicStyle = {...wrapStyle, color: undefined, fontStyle: 'italic', fontWeight: undefined}
-const strikeStyle = {...wrapStyle, color: undefined, fontWeight: undefined, textDecoration: 'line-through'}
+const textBlockStyle = {display: 'block', ...wrapStyle, color: 'inherit', fontWeight: 'inherit'}
```

Comment:
using undefined is problematic here. it caused issues on mobile and it changes how things are merged (works with object merging but not using glamor.css(myStyles, textBlockStyle)